Test Project/Assets/Scripts/ToggleButtonColour.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class ToggleButtonColour : MonoBehaviour {
+
+
+	#region Serialized Fields
+	
+	[SerializeField]
+	private List<Color> m_Colors;
+	[SerializeField] 
+
+	private Button m_TargetButton;
+
+	#endregion
+
+
+	#region Non-Serialized Fields
+
+	[System.NonSerialized]
+	private int m_NextColor = 1;
+
+	#endregion
+
+
+	#region Monobehavior Methods
+
+	private void Start()
+	{
+		SetColor(0);
+	}
```

Comment:
Looks like the auto-formatting gremlins snuck back in, as I see this in a few places in the file. Opening brackets should be on the same line.